packages/core/src/filters/defaultFilter.vert

```diff
@@ -2,25 +2,27 @@ attribute vec2 aVertexPosition;
 
 uniform mat3 projectionMatrix;
 
-varying vec2 vTextureCoord;
+varying vec2 vTextureCoord;// actually the input texture coord (gl_FragColor to set output)
 
 uniform vec4 inputSize;
-uniform vec4 outputFrame;
```

Comment:
Same problem.

Reply:
Yeah, I forgot about that. Maybe we can make them inputFrame and outputFrame?

Reply:
We can rename it, consistently, but we must found their usage on core and pixi-* packages too.
It isn't simple, but possible.

Reply:
If I use inputFrame, then we don’t need to rename outputFrame and it will be consistent, right?

Reply:
I changed them.